core/templates/components/question-directives/question-editor/question-editor.component.html

```diff
@@ -10,7 +10,8 @@
                       (onSaveInapplicableSkillMisconceptionIds)="saveInapplicableSkillMisconceptionIds($event)"
                       [interactionIsShown]="interactionIsShown"
                       [stateContentPlaceholder]="getStateContentPlaceholder()"
-                      [stateContentSaveButtonPlaceholder]="getStateContentSaveButtonPlaceholder()">
+                      [stateContentSaveButtonPlaceholder]="getStateContentSaveButtonPlaceholder()"
+                      [isSticky]="isSticky">
```

Comment:
We try to avoid starting variables with verbs. I think you can name this something like stateContentShouldStayVisibleOnScroll or similar. Similarly elsewhere.